packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-pwd-recovery-mfa/main/nodejs/integrationsteps.md

```diff
@@ -1,121 +1,122 @@
+### 3. Submit username
 
-The call is shown in the `recover-password.js` file of the SDK sample application.
+The user enters their username on the password recovery page and selects **Next** to start the recovery process.  At this time, call `OktaAuth.idx.recoverPassword()`, passing in the username.
 
-```JavaScript
-router.post('/recover-password', async (req, res, next) => {
-  const { username } = req.body;
-  const authClient = getAuthClient(req);
+```javascript
+try {
   const transaction = await authClient.idx.recoverPassword({ username });
   handleTransaction({ req, res, next, authClient, transaction });
-});
+} catch (error) {
+  next(error);
+}
 ```
 
-The `handleTransaction.js` page routes the user to a select authenticators page (`/select-authenticator` in this example) that the user needs to use to select an authentication type. In this case, the email factor is configured.
-
-```JavaScript
-// authenticator authenticate
-    case 'select-authenticator-authenticate':
-      redirect({
-        req, res, path: '/select-authenticator'
-      });
-      return true;
+### 4. Display a list of available authenticators
+
+`OktaAuth.idx.recoverPassword()` returns a response indicating that the next step is to select an authenticator. Specifically, `IdxTransaction` returns:
+
+* a `status` of `PENDING`
+* a `nextStep.name` equal to `select-authenticator-authenticate`
+* an `option` item with a `value` of `okta_email` in the `nextstep.inputs[n].options` array
+
+```javascript
+{
+  status: "PENDING",
+  nextStep: {
+    name: "select-authenticator-authenticate",
+    inputs: [
+      {
+        name: "authenticator",
+        type: "string",
+        options: [
+          {
+            label: "Email",
+            value: "okta_email",
+          },
+          {
+            label: "Phone",
+            value: "phone_number",
+          },
+        ],
+      },
+    ],
+  },
+}
 ```
 
->**Note:** Review the complete use of `idx.recoverPassword` in the [Okta Auth JS SDK](https://github.com/okta/okta-auth-js/blob/master/docs/idx.md#idxrecoverpassword).
-
-### 2: Create the reset password authenticators page
-
-The next step is to create a page that shows the authenticator key returned from the `handleTransaction()` method. For this use case, the email authenticator is returned (`authenticator: AuthenticatorKey.OKTA_EMAIL`). The page includes the name of the returned authenticator and the ability to select the authenticator to initiate the authentication verification process.
+Using this response, display a list of authenticators to the user. If you've configured your org as described in [Configuration updates](#configuration-updates), email is included in the list of available authenticators.
```

Comment:
Splitting this out into a note. Your code demonstrates that email is in the collection so the instruction should just be "display it" now. The other sentence is an aside that should go under the image.

```suggestion
Using this response, display a list of authenticators to the user. 

> **Note**: If email is not included in the list of available authenticators, check you've configured your org as described in [Configuration updates](#configuration-updates).
```

Reply:
The aside makes sense. I'm curious though why you switched the text to using "not". Let's talk about this tomorrow.

Reply:
updated.